docs/04coin-set-model/intro.md

```diff
@@ -55,4 +55,4 @@ Value may only be added to the coin set via the pre-farm (a one-time occurrence)
 Typically, in a block's combined spend bundle, value added will be equal to value spent, other than the block rewards. By definition, there are two possible exceptions:
 
 * Value added > value spent -- This is not allowed, so the transaction will be rejected. The rejection will usually happen at the mempool level, though a malicious actor could write their own mempool to accept the transaction, in which case the blockchain will reject it.
-* Value added < value spent -- This is allowed, so the transaction will succeed. In this case, the remaining value will be rewarded to the farmer of the block containing the transaction.
\ No newline at end of file
+* Value added =< value spent -- This is allowed, so the transaction will succeed. If the value added is less than the value spent the remaining value will be rewarded to the farmer of the block containing the transaction awarded as a tip (you probably don't want to do that!).

```

Comment:
If you think the extra wording is needed for clarification, that's fine. However, I don't know what "=<" means.

Reply:
Less than or equal to.

Reply:
If the intention was for it to mean "less than or equal to" then that's incorrect. If the two values are equal, then that's the typical case. The two bullet points are the exceptions, when the values are not equal.